test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -161,10 +163,11 @@ TEST_F(OutlierDetectorImplTest, DestroyHostInUse) {
 
   detector.reset();
 
-  loadRq(cluster_.hosts_[0], 5, 503);
+  loadRq(cluster_.hosts_[0], 5, 500);
 }
 
 TEST_F(OutlierDetectorImplTest, BasicFlow5xx) {
+
```

Comment:
new del newlines like these. Also, please add small comments on top of the new tests for what the test does (we are trying to do this for new tests).